app/controllers/patients_controller.rb

```diff
@@ -119,7 +116,7 @@ def find_patient
                   :procedure_cost, :check_number, :date_of_check]
   ].freeze
 
-  OTHER_PARAMS = [:urgent_flag, :initial_call_date, :pledge_sent].freeze
+  OTHER_PARAMS = [:urgent_flag, :initial_call_date, :pledge_sent, :last_edited_by].freeze
```

Comment:
`last_edited_by` we draw from the `current_user` variable in the controller, not the payload params, so we can revert this change.

Reply:
Good point!